test/index.js

```diff
@@ -2302,22 +2302,21 @@ let AllTests = ContextureClient => {
         key: 'root',
         join: 'and',
         children: [
-          { key: 'pivot', type: 'pivot', groups },
+          { key: 'pivot', type: 'pivot', drilldown: [], rows },
           { key: 'test', type: 'facet', values: [] },
         ],
       }
     )
 
     // These tests set `forceReplaceResponse` conditionally during mutate based on the pivot's onDispatch
```

Comment:
It looks like this comment needs to be updated.

Reply:
Thank you! I will update it in my next PR